core/modules/czar/Czar.h

```diff
@@ -104,6 +104,7 @@ class Czar {
     std::unique_ptr<ccontrol::UserQueryFactory> _uqFactory;
     ClientToQuery _clientToQuery;       ///< maps client ID to query
     std::mutex _mutex;                  ///< protects both _uqFactory and _clientToQuery
+
```

Comment:
I guess this is leftover from you experiments with adding data members here, may be better not to touch this file at all in this commit?